generators/swift/codegen/src/ast/Func.ts

```diff
@@ -0,0 +1,63 @@
+import { AstNode, Writer } from "@fern-api/generator-commons";
+import Swift, { AccessLevel, FunctionModifier, Param } from "../swift";
+
+export declare namespace Func {
```

Comment:
(not a blocker): was really valuable to add doc strings in the ast at the function level, class level, parameter level